docs/sql-ref-syntax-qry-select-window.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: global
+title: WINDOW Clause
+displayTitle: WINDOW Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+**This page is under construction**
```

Comment:
Is there value in adding these placeholders, vs just adding/linking them when available?

Reply:
@srowen Thanks. Actually, thats the approach we followed when we started this work a few months back. I guess we didn't want to have a link broken at any point. Also, makes it easier for others to contribute without having to rebase ?

Reply:
Yeah I know, I'm not sure that was great as not all are filled out. Unless it would be hard to link them everywhere later, I wonder why they need to be here now? I don't see a difference w.r.t merging. I don't feel strongly but think we are just going to end up with more dummy pages that nobody fills out

Reply:
@srowen OK. Let me remove the links and push.